src/components/navigation/problem-tab-content.tsx

```diff
@@ -36,6 +36,10 @@ export const ProblemTabContent: React.FC<IProps>
   const problemsPanelHeight = vh - headerOffset;
   const problemsPanelStyle = { height: problemsPanelHeight };
 
+  if (ui.showChatPanel) {
+    ui.updateFocusDocument();
```

Comment:
This is going to trigger an update on every render which seems excessive. If you look at other callers of `updateFocusDocument()` they're all in response to user action. There must be a better way to determine when/whether such a refresh is required.

On an unrelated point that I just noticed, line 35 above looks wrong: `kHeaderHeight + + kNavTabHeight`?